export const throwError = async <T>(): Promise<T> => {
  throw new Error()
}
